mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/samples/links/createTwoLinks.json

```
@@ -1,6 +1,7 @@
 {
   "links": [
     {
+      "linkingRuleId": 7,
```

Should be strict with linking rules /linking-rules/instance-authority
```suggestion
      "linkingRuleId": 12,
```